build/version.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-commonjs, import/no-nodejs-modules */
+
+const execSync = require('child_process').execSync;
+const prependFile = require('prepend-file');
+const metadata = require('../package.json');
+
+const unreleasedChangelog = 'changelog/UNRELEASED.md';
+const templateChangelog = 'changelog/_EXAMPLE.md';
+
+const version = metadata.version;
```

Comment:
You should just be able to use `process.env.npm_package_version` here. See: https://docs.npmjs.com/misc/scripts#packagejson-vars